docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - 5432:5432
+  web:
+    image: crimethinc:latest
+    ports:
+      - "3000:3000"
+    links:
+      - db
+    environment:
+      - RAILS_ENV=docker
```

Comment:
For a first draft, this is gonna be fine, tho. But I think we really need to find a way to run every environment in docker. You should probably check this [PR](https://github.com/rails/rails/pull/46762). In rails 7.1 that is gonna be release soon, DHH add the possibility to generate a default `Dockerfile`.